src/activities/content/ContentWebLinkEntity.js

```diff
@@ -61,6 +61,18 @@ export class ContentWebLinkEntity extends ContentWorkingCopyEntity {
 		return this._entity && this._entity.properties && this._entity.properties.url;
 	}
 
+	type() {
```

Comment:
nit: up to you, but I would rename this to getWebLinkType()

Reply:
Normally I would agree, but I feel `type()` fits in better with what is currently here (most of these entities have getters that are just `propName()`

Reply:
Do we wanna add a comment about the return context as we do for other properties?